libs/utils/decorate.js

```diff
@@ -25,21 +25,32 @@ export function decorateButtonsGroup(el) {
 }
 
 export function decorateBlockText(el, size = 'small') {
+  const blockTypeSizes = {
+    media: {
+      //name: [heading, detail, body]
+      small: ['XS', 'M', 'S'],
+      medium: ['M', 'M', 'S'],
+      large: ['XL', 'L', 'M'],
+      xlarge: ['XXL', 'L', 'M'],
+    },
+    text: {
+      small: ['M', 'S', 'S'],
+      medium: ['L', 'M', 'M'],
+      large: ['XL', 'L', 'M'],
+      xlarge: ['XXL', 'XL', 'L'],
+    }
+  };
+  const sizeType = el.classList.contains('text-block') ? blockTypeSizes.text[size] : blockTypeSizes.media[size];
+  const decorateHeading = (headingEl, sizes) => {
+    headingEl.classList.add(`heading-${sizes[0]}`);
```

Comment:
Can we have the heading as an optional requirement? Currently if you remove the heading, the block fails to load. There are instances where an author may want a quick button, centered on the page and the text block could be a good path to that.

Reply:
@fullcolorcoder  - Love it. Yes. I'll update the logic to not depend on a headline. I'll let you know when/where that can be tested.

Reply:
@fullcolorcoder  - I have actually already accounted for this now that i'm looking at it closer. That decorateHeading() only gets used if there is a heading. 
See [this page](https://text-util-grid--milo--adobecom.hlx.page/drafts/rparrish/text-more) to test the use case you're describing.